sbndcode/BlipRecoSBND/Alg/BlipRecoAlg.cc

```diff
@@ -1072,7 +1072,7 @@ namespace blip {
               TVector3 p2(b.X(), b.Y(), b.Z() );
               // TO-DO: if this track starts or ends at a TPC boundary, 
               // we should extend p1 or p2 to outside the AV to avoid blind spots
-              TVector3 bp = newBlip.Position;
+              TVector3 bp(newBlip.Position.X(), newBlip.Position.Y(), newBlip.Position.Z());
```

Comment:
Conversion `geo::Point_t` → `TVector3`:
```suggestion
              TVector3 bp = geo::vect::toTVector3(newBlip.Position);
```
(`#include "larcorealg/Geometry/geo_vectors_utils_TVector.h"`); or using the more generic `convertTo()`:

```suggestion
              TVector3 bp = geo::vect::convertTo<TVector3>(newBlip.Position);
```
(for this one: `#include "larcorealg/Geometry/geo_vectors_utils.h"`).